site/pages/concepts/middleware.mdx

```diff
@@ -17,11 +17,11 @@ When using the [React](/react/overview), [Core](/core/overview), or [Infra](/inf
 If you want to sponsor gas, the Smart Account Clients in these packages abstract the relevant middleware away from you since all you need is a `policyId` to sponsor gas with Alchemy.
 :::
 
-As mentioned abve, the client can be configured with a series of middleware that always run in the same order:
+As mentioned above, the client can be configured with a series of middleware that always run in the same order:
```

Comment:
the rest of the changes are just fixing typos that i noticed while figuring out my way around the codebase...